annlite/executor.py

```diff
@@ -58,6 +62,11 @@ def __init__(
         self.index_traversal_paths = index_traversal_paths
         self.search_traversal_paths = search_traversal_paths
         self._valid_input_columns = ['str', 'float', 'int']
+        self._task_queue = DocumentArray()
+        self._index_batch_size = 1024
```

Comment:
This hyper parameter can be automatically determined by the context (e.g., the number of CPU cores).

Reply:
ignore my question above.